website/docs/style/design-tokens/design-tokens.md

```diff
@@ -96,54 +96,34 @@ const App = DesignTokens;
 
 ## Themes
 
-If you are creating a product that differs in style from other products of Semrush, apply a custom theme for your branding.
+If you need to build a product that differs in style from our default theme, you can create a custom theme for it.
 
-### What is a theme?
+Theme is a set of tokens represented in CSS variables that has different from the default sets values. Values can be changed for base, semantic, or both sets. You can redefine them globally or only for a specific subtree of React app. Refer to [Usage in development](/style/design-tokens/design-tokens-usage-development).
 
-Theme is a set of design tokens represented in CSS variables that differs from the default sets (base or semantic, or both). You can redefine them globally or only for a specific subtree of React app. Refer to [Usage in development](/style/design-tokens/design-tokens-usage-development).
-
-## Creating your own theme
-
-**First of all, answer these questions for your case:**
-
-1. Do you want to use the components of the Intergalactic Design System and need a completely different look and feel for them?
-2. Is your product not a Semrush core product?
-
-If the answer to all the questions above is yes, then you definitely need a theme that is different from the default one.
+## Creating new theme
 
 ### Step one. Design new theme
 
 Creating a theme usually starts with design. In fact, this is the most time-consuming part.
 
-There are several ways (and, of course, there are more than the two that we offer next!). You can choose the one that suits you best based on your case.
-
-Let's take a look at the two main theme creation situations: local and global.
+**We recommend you using the native Figma variables (tokens) functionality**. But if you need more functionality that Figma doesn't have at the moment, use [Tokens Studio plugin for Figma](https://www.figma.com/community/plugin/843461159747178978). It's one of the most powerful tools for managing tokens, linking styles between the code and Figma files.
 
-#### **Local theme: you need to try on other tokens or a new theme for a component or part of the design system**
+Refer to the following tutorials, for the detailed process for creating a new theme:
 
-In this case we recommend you using [Tokens Studio plugin for Figma](https://www.figma.com/community/plugin/843461159747178978). It's one of the most powerful tools for managing tokens, linking styles between the code and Figma files, and it can help you save time trying new values for tokens of your new theme.
+- Internal libraries have two tutorials, [tutorial for Figma tokens](https://www.figma.com/design/1TV7YbEL3FaV0znCkQtsrC/Themes'-playground-%26-tutorial-%F0%9F%8E%93?node-id=13125-73031&node-type=canvas&t=qOTf0DSn0M8p63of-11) and [tutorial for Tokens Studio](https://www.figma.com/design/K1s6wF8NTH3uNHvjkn6hjc/Themes'-playground-%26-tutorial-%F0%9F%8E%93?m=auto&t=jHrLhhOMB32IMklB-6)
```

Comment:
```suggestion
- Semrush designers and developers can use two tutorials, [tutorial for Figma tokens](https://www.figma.com/design/1TV7YbEL3FaV0znCkQtsrC/Themes'-playground-%26-tutorial-%F0%9F%8E%93?node-id=13125-73031&node-type=canvas&t=qOTf0DSn0M8p63of-11) and [tutorial for Tokens Studio](https://www.figma.com/design/K1s6wF8NTH3uNHvjkn6hjc/Themes'-playground-%26-tutorial-%F0%9F%8E%93?m=auto&t=jHrLhhOMB32IMklB-6)
```
might be easier to understand if we say "for Semrush designers...", instead of "internal libraries", someone might get confused what's the connection between libraries and tutorials, or something like that :)